aggregator_core/src/datastore.rs

```diff
@@ -5722,6 +5733,9 @@ struct TaskInfo {
 
     /// The task's report expiry age, corresponding to the tasks.report_expiry_age column.
     report_expiry_age: Option<chrono::Duration>,
+
+    /// The task's time precision
+    time_precision: Duration,
```

Comment:
Hmm, mixing `chrono::Duration` and `janus_messages::Duration` in a struct is a little smelly. But I don't think it's worth fixing here. We have an issue backlogged for not using on-the-wire time types in business logic.